pyout/tabular.py

```diff
@@ -144,6 +145,13 @@ class Tabular(interface.Writer):
     def __init__(self, columns=None, style=None, stream=None,
                  interactive=None, mode=None, continue_on_failure=True,
                  wait_for_top=3, max_workers=None):
+        in_jupyter = "JPY_PARENT_PID" in os.environ
```

Comment:
Whatever works!  Just FTR 

in DataLad we use another (probably not better) approach and rely on having `get_ipython()` in the environment (https://github.com/datalad/datalad/blob/master/datalad/utils.py#L262  ).  That allows to see if we are in terminal ipython
```
In [1]: get_ipython()                                                                                            
Out[1]: <IPython.terminal.interactiveshell.TerminalInteractiveShell at 0x7ff8ccb23550>
```
which is ok for pyout interactive display, or notebook where it returns `<ipykernel.zmqshell.ZMQInteractiveShell at 0x7fa504be7700>`

And I see that in the terminal ipython shell there is no env var set
```
In [4]: os.environ['JPY_PARENT_PID']                                                                             
...
KeyError: 'JPY_PARENT_PID'
```
so similar desired effect would be here.

Reply:
Thanks for taking a look.

> in DataLad we use another (probably not better) approach and rely on having `get_ipython()` in the environment

I'm not sure I understand your comment, but I don't see how relying on `get_ipython` being defined for a `!command` that ends up executing pyout would work.

> And I see that in the terminal ipython shell there is no env var set

Good to see.  As I mentioned in bf5743a1, this variable discriminates a notebook from an IPython terminal on my end too.

Reply:
> being defined for a `!command` that ends up executing pyout would work.

ah indeed -- it would not indeed, good point.  It would work only for direct use of pyout in ipython notebook. 

So all is good, lets proceed! ;)